homeassistant/components/google_assistant/report_state.py

```diff
@@ -40,8 +40,27 @@ async def report_states(now=None):
 
         # We will report all batches except last one because those are finalized.
         while len(pending) > 1:
+            # Turn the last_event attribute into a notification
+            # https://developers.home.google.com/cloud-to-cloud/integration/notifications
+            states_notifications = pending.popleft()
+            states = {
+                key: {attr: val for attr, val in values.items() if attr != "last_event"}
+                for key, values in states_notifications.items()
+            }
+            notifications = {
+                key: {
+                    "ObjectDetection": {
+                        "objects": {"unclassified": 1},
+                        "priority": 0,
+                        "detectionTimestamp": timestamps["last_event"],
+                    }
+                }
+                for key, timestamps in states_notifications.items()
+                if "last_event" in timestamps
+            }
+            notifications["notificationSupportedByAgent"] = True
```

Comment:
Ah so this is the wrong place?

Reply:
Yup: should be enabled here: https://github.com/home-assistant/core/blob/0a255d000772234011a51a8e0ba057456bb00b35/homeassistant/components/google_assistant/helpers.py#L614 if there is a trait with notification support.

Reply:
Thnx, I'll have a look

Reply:
Moved the setting. Now there is an extra option to enable announcements, so that works now.
What still does not work is that sending notification should trigger the announcement. Any idea?

Reply:
I think you must allow it in the google home app on the device. I need to look more closely on the data that is being sent too.

Reply:
That is correct, when I moved it to `helpers.py` as you suggested I was able to allow it in the google home app. But it might be possible to insert this from the trait.